doc/api/cli.md

```diff
@@ -1116,6 +1116,27 @@ unless either the `--pending-deprecation` command-line flag, or the
 are used to provide a kind of selective "early warning" mechanism that
 developers may leverage to detect deprecated API usage.
 
+### `--permission-fs-path-delimiter`
+
+<!-- YAML
+added: v20.0.0
+-->
+
+> Stability: 1 - Experimental
+
+This flag configures file system path delimiter for permissions using
+the [Permission Model][].
+
+Examples can be found in the [File System Permissions][] documentation.
+
+Especial characters in bash as `;` must be escaped or quoted:
```

Comment:
```suggestion
Special shell characters such as as `;` must be escaped or quoted:
```

Reply:
I think there is an extra unneeded as "such as as"